### PR TITLE
tesseract: update to 5.4.1

### PR DESCRIPTION
**Pull request by BC204**
Topic Description
-----------------

- skanpage: bump REL for topic Revision Marking Guidelines
- crow-translate: bump REL for topic Revision Marking Guidelines
- opencv: bump REL for topic Revision Marking Guidelines
- tesseract: bump REL for topic Revision Marking Guidelines
- tesseract: update versions in PKGBREAK
- crow-translate: bump REL due to tesseract update
- skanpage: bump REL due to tesseract update
- opencv: bump REL due to tesseract update
- tesseract: update to 5.4.1
    - Fix the wrong anitya id
    - modify path of tesseract_fast in beyond

Package(s) Affected
-------------------

- crow-translate: 2.11.1-2
- opencv: 4.9.0-4
- skanpage: 23.08.5-2
- tesseract: 5.4.1-1

Security Update?
----------------

No

Build Order
-----------

```
#buildit tesseract opencv crow-translate skanpage
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [x] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

### CODE REVIEW

**Comment by BC204** [comment]
Test on amd

Example picture
musicbox-0 3 1

Test result
tesseract-5 4 1

**Review by MingcongBai — CHANGES_REQUESTED** [review]
Wow, what a soname...

```
 0x000000000000000e (SONAME)             Library soname: [libtesseract.so.5.4.1]
```

However, according to the [Packages site](https://packages.aosc.io/revdep/tesseract), you seem to be missing rebuilds. Please also update the PKGBREAK line in `defines`.

File: app-utils/tesseract/autobuild/beyond [hunk 1]
```diff
@@ -1,7 +1,7 @@
 abinfo "Packing trained data..."
 mkdir -vp "$PKGDIR"/usr/share/tessdata
 
-pushd ../tessdata_fast-4.1.0
+pushd ../tessdata_fast
```
Comment (MingcongBai) [inline app-utils/tesseract/autobuild/beyond:4]: This should be `"$SRCDIR"/../tessdata_fast`.

**Review by MingcongBai — CHANGES_REQUESTED** [review]
Tested good on LoongArch

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**